packages/angular/.storybook/public/assets/demo.css

```diff
@@ -0,0 +1,181 @@
+@import './../../../../dist/clr-core/styles/module.reset.css';
```

Comment:
Majority of these styles are core demo specific so not sure if they are needed for the Angular demo examples

Reply:
yeah, thats something I need to clean up from the POC I started with. Will remove.